events/factories.py

```diff
@@ -0,0 +1,33 @@
+import factory
+from django.utils import timezone
+
+from fair.factories import FairFactory
+from .models import Event
+
```

Comment:
Basically, according to PEP8:

1. Standard library imports
2. Third party imports
3. Local application imports

With a blank line between each group.